base/io.jl

```diff
@@ -285,7 +304,7 @@ end
 
 readline() = readline(STDIN)
 readline(s::IO) = readuntil(s, '\n')
-readchomp(x) = chomp!(readall(x))
```

Comment:
While we're at it, maybe we could deprecate `readchomp` in another commit? Code isn't that longer without it...